### ISSUE TITLE
Remove reference to Monopoly in the recovery key dialog

### ISSUE THREAD

**Issue by GTcreyon** [Nheko-Reborn/nheko#1637]
### The Problem

When being issued a recovery key, there's a joke after the explanation reading something like "Do not pass go. Do not collect $200."

Look, I appreciate the humor, but we're moving into a new generation of users who have never touched Monopoly and would not have a clue what this was supposed to mean. Combine that with the confusing language and involving finances in the mix, that seems needlessly confusing and anxiety-inducing.

There's some dissonance between "Here's your recovery key, you absolutely cannot lose this" and "Here's a joke about a game where you lose everything you own and you don't talk to your aunt anymore after what happened over the board at last Thanksgiving."

### The Solution

Just remove those extra sentences. Everything else is fine.

### Happens in the latest version

- [X] Yes, this feature is still missing.

**Comment by deepbluev7** [comment]
I mean, I understand the concern, but it would be very boring. Also you shouldn't sell your recovery key either, the joke is there to make those important lessons more memorable and so far that seems to work out well. So for now I won't be doing this, sorry. We are not serious enough of a project for this!

**Comment by GTcreyon** [comment]
> it would be very boring

I think "boring" is desirable when we're talking security. The opposite, call it "fun" isn't really what I'd associate with a security setup. If I said "I did something **fun** with your login details today!", you would probably want to think about changing your password.

Boring isn't inherently bad, and excitement isn't necessarily good. Case study: Opera GX's latest marketing campaign that resulted in a pretty solid PR hit. You gotta be sensitive about it to decide when and how it's appropriate, and I just don't think this is it.

> the joke is there to make those important lessons more memorable

I can see the value in that, but I feel like if you're going to have a joke here, it should be less esoteric. I'm not against the idea of there being a joke - and you make a good point with the educative power of humor - but I think this could be conveyed in a way that is easier to understand.

Jokes do tend to be funnier when they're niche, sure. But if somebody doesn't get the joke (e.g. if they've never played Monopoly in English), it's going to be a net detriment. And, if the point of the joke is to help people learn, then this is no good.

> We are not serious enough of a project for this!

You don't need to be a serious project to keep your tutorial strings sober. Making the software more accessible and easier to understand doesn't make you corporate or boring, it's just considerate that not everyone is going to be perceptive of your sense of humor. I'm autistic, for example. I get it, I can process "oh this is a monopoly joke eheh," but ultimately it just adds an extra layer of figuring-out to something that should be as straightforward as possible.

Besides, there's plenty of room elsewhere in the software to sprinkle in some humor. I feel like the security key is just about the highest-risk place you could put it, no?

In summary: I don't personally like the inclusion of a joke here, but I can see its value. That said, if you want to make a joke there, I'd advise a different one that more people can understand and doesn't involve money (or Monopoly money).
